pages/1_14/getting_started/new_project.md

```diff
@@ -0,0 +1,167 @@
+
+    // Other stuff
+}
+```
+
+In this section you will add any dependencies your project may have.
+At the beggining the only thing that is added here is the forge version.
+
+If you ever want to update the version of
+Forge within your mod, simply update this value.
+
+You may put jars on which you depend on in ./libs or you may define them like so:
+```groovy
+compile "some.group:artifact:version:classifier"
+compile "some.group:artifact:version"
+```
+
+The 'provided' configuration is for optional dependencies that exist at compile-time but might not at runtime.
+```groovy
+provided 'com.mod-buildcraft:buildcraft:6.0.8:dev'
+```
+
+For more info:
+https://docs.gradle.org/current/userguide/dependency_management_for_java_projects.html
+
+### Jar manifest
+This last section of the build.gradle allows you to add
+values to your output jar's manifest. The default
+`build.gradle` adds some values about the author and mod here.
+
+```groovy
+jar {
+    manifest {
+        attributes([
+            "Specification-Title": project.name, // This may be different by default but you can change it to project.name
+            "Specification-Vendor": "author",
+            "Specification-Version": "1", // We are version 1 of ourselves
+            "Implementation-Title": project.name,
+            "Implementation-Version": "${version}",
+            "Implementation-Vendor" :"author",
+            "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
+        ])
+    }
+}
+```
+
+As you can see the date, name, and version have been taken care of.
+All you have to do is enter your name in the places of `author`
+
+### Using the gradle.properties
+
+To make life easier, you don't have to keep searching through the build.gradle for a few values to change any time you
+want to update a version. You can make use of your `gradle.properties` file to collect variables in. You can check out
+an example of that in use here in Sparks Hammers:
+* [build.gradle](https://github.com/thebrightspark/SparksHammers/blob/1.12/build.gradle)
+* [gradle.properties](https://github.com/thebrightspark/SparksHammers/blob/1.12/gradle.properties)
+
+!! NOTE !!
+Although the above examples are from a 1.12 mod, Gradle still works the same in this case.
+
+### Setting up your workspace
+
+Once you have everything extracted out of the MDK and configured for your project, you can now build your workspace.
+
+All you have to do is import the project into your IDE of choice
+and generate the launch tasks for your IDE.
+
+* In Intellij Idea you can do this using the `File -> Open...` menu option and selecting your project directory. To generate to tasks you
+have to run `gradlew genIntellijRuns`
+
+!! NOTE !!
+
+You might need to manually set the module to `MODNAME.main` in your
+launch configurations
+
+* For Eclipse you have to run `gradlew genEclipseRuns`
+
+### Building your mod
+
+Once you're ready to build your mod, you can simply run `gradlew build`.
```

Comment:
As above, if you've imported the Gradle project in IDEA, you can use the Gradle window to run the task